code/modules/projectiles/guns/projectile/toy.dm

```diff
@@ -75,6 +75,10 @@
 	if(chambered && !chambered.BB)
 		qdel(chambered)
 
+/obj/item/gun/projectile/shotgun/toy/process_fire(atom/target as mob|obj|turf, mob/living/user as mob|obj, message = 1, params, zone_override, bonus_spread = 0)
```

Comment:
I kept the ``as`` stuff for sake of consistency but it's not needed, happy to cut it if requested

Reply:
`mob/living/user as mob|obj` I dunno if that's right bud

Reply:
removed